app/src/test/java/ValidatorTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.Test;
+
+import teamagam.ConnectionTest.HttpValidator;
+import teamagam.ConnectionTest.PingValidator;
+import teamagam.ConnectionTest.Validator;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class ValidatorTest {
+
+    private static final String RELIABLE_SERVER_ADDRESS = "google.com";
+    private static final String UNRELIABLE_SERVER_ADDRESS = "Mamas42Empire";
+
+    @Test
+    public void workingPingTest() {
+        Validator pingValidator = new PingValidator();
```

Comment:
You can extract the two Validators as members of the test class and init them on a setup method (annotated @Before)